package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "3.4.9",
+  "version": "3.4.10",
```

Comment:
You need to update line 77 in this file from `git:` to `https:`.

Reply:
Updated